D.sublime-build

```diff
@@ -7,31 +7,32 @@
   [
     {
       "name": "Run",
-      "cmd": ["rdmd", "-g", "-debug", "$file"]
+      "working_dir": "$file_path",
+      "cmd": ["rdmd", "-g", "-debug", "$file_name"]
     },
     {
       "name": "dub",
-      "working_dir": "$project_path",
```

Comment:
dub must be run from the directory containing its package file, AFAIK.
I think using @zeroows solution in this [PR](https://github.com/yazd/DKit/pull/28) is more suitable due to that.

I agree with your changes for rdmd, and single file builds.